tests/10-EdvClient.spec.js

```diff
@@ -1,6 +1,8 @@
 /*!
  * Copyright (c) 2018-2019 Digital Bazaar, Inc. All rights reserved.
  */
+'use strict';
```

Comment:
Can remove all these `'use strict';` pragmas from ES6+ code.

Reply:
Removed them from the test specs: https://github.com/digitalbazaar/edv-client/pull/26/commits/2f851b4f632828632ca44c6f5bcd8cf2712abd83